### DISCUSSION TITLE
npm run build

### DISCUSSION THREAD

**Discussion by Joaoof** [adonisjs/core#4220] [Help]
I'm having problems when running the build in the dockerfile: 
![Captura de tela 2023-08-14 144237](https://github.com/adonisjs/core/assets/113441117/bb3aab70-9a46-4844-b59e-f531030cd211)
my .tsconfig: 
![Captura de tela 2023-08-14 144332](https://github.com/adonisjs/core/assets/113441117/857fc6d4-74f7-407d-b44f-6915d9609f19)
package.json: 
![Captura de tela 2023-08-14 144435](https://github.com/adonisjs/core/assets/113441117/6007da57-28a6-4538-839f-c2b34c35fab7)

**Comment by MaximeMRF** [comment]
Hello @Joaoof

I personally ignore ts errors

```dockerfile
RUN node ace build --production \
    --ignore-ts-errors
```

**Reply by Joaoof** [reply]
Hello, 
I already tried that option, and it didn't work.

**Comment by RomainLanz** [comment]
It seems that you have some syntax errors.
It would be way easier to help if you give us the exact error you are having.

**Comment by Joaoof** [comment]
error resolvido
